.github/workflows/release.yml

```diff
@@ -126,10 +126,11 @@ jobs:
           cd ../..
 
           npm install --global @bytecodealliance/jco@$JCO_VERSION
-          jco transpile $WASM_FILE -o typegraph/deno/src/gen
-          deno run -A typegraph/deno/dev/fix-declarations.ts
+          jco transpile $WASM_FILE -o typegraph/node/src/gen
 
-          cd typegraph/deno/src
+          cd typegraph/node
+          pnpm run build
+          cd dist
           deno run -A ../../../dev/publish.ts
```

Comment:
dev/publish.sh is not needed anymore, and we have to integrate its content into the package.json, pnpm publish should be enough for the node side

Reply:
fixing it